chain/arweave/src/adapter.rs

```diff
@@ -56,17 +62,35 @@ impl ArweaveTransactionFilter {
                     && !data_source.mapping.transaction_handlers.is_empty()
             })
             .map(|ds| {
-                base64_url::decode(&ds.source.owner.clone().unwrap_or_default()).unwrap_or_default()
+                let owner = ds.source.owner.clone().unwrap_or_default();
```

Comment:
`clone` could be made only `owner.len() == HEX_SHA256_LEN`, the other branch of the if does not need an owning copy.

Reply:
this was fixed in a differnet PR